resources/styles/global/tutor-booksplash-background.less

```diff
@@ -0,0 +1,44 @@
+// Applies a fixed background effect to a container that includes a large fixed header
+//
+// It's intended to be customized for different looks depending on the type of content
+// the user is styling, i.e. Physics, Biology, etc
+.tutor-booksplash-background {
+
+  &:before {
+    content: attr(data-title);
+    position: fixed;
+    left: 0px;
+    right: 0px;
+    overflow: hidden;
+    background-color: @tutor-tertiary;
+    color: @tutor-secondary;
+    text-align: center;
+    font-size: 300px;
+    line-height: 200px;
+    font-family: 'HelveticaNeue-Bold', 'Helvetica Neue Bold', 'Helvetica Neue';
```

Comment:
Since Helvetica is not included as a font I think it should either:

1. be included in `resources/fonts` (assuming we have rights to use it) and include it in `typography.less`
2. include a different font